packages/uiweb/src/lib/components/space/SpaceTrendingList.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useContext, useEffect } from 'react';
+import { useSpaceData } from '../../context';
+
+export interface SpaceTrendingListProps {
+  // Add props specific to the SpaceTrendingList component
+  temporary?: string; // just to remove eslint error of empty prop
+}
+
+const SpaceTrendingList: React.FC<SpaceTrendingListProps> = () => {
+  const { trendingListData, setTrendingListData } = useSpaceData();
+
+  useEffect(() => {
+    // Simulating an API call to fetch trending list data
+    const fetchTrendingListData = async () => {
+      try {
+        // Make an API call to fetch the trending list data
+        // const response = await 
+        // const data = ;
+
+        // Update the trending list data in the context
+        // setTrendingListData(data);
+      } catch (error) {
+        console.error('Error fetching trending list data:', error);
+      }
+    };
+
+    // Fetch trending list data if it's not already available
+    if (!trendingListData) {
+      fetchTrendingListData();
+    }
+
+    // Set up a timer to fetch updated data every 2 minutes
+    const timer = setInterval(fetchTrendingListData, 2 * 60 * 1000);
```

Comment:
suggestion: we should write a custom poller for polling the API. `setInterval` approach has a drawback, that if the API takes too long to respond, it will be stuck and will block JS.

can implement this later as well, not a prio